prow/kube/prowjob.go

```diff
@@ -79,22 +79,45 @@ type ProwJob struct {
 }
 
 type ProwJobSpec struct {
-	Type    ProwJobType  `json:"type,omitempty"`
-	Agent   ProwJobAgent `json:"agent,omitempty"`
-	Cluster string       `json:"cluster,omitempty"`
-	Job     string       `json:"job,omitempty"`
-	Refs    *Refs        `json:"refs,omitempty"`
-
+	Type      ProwJobType  `json:"type,omitempty"`
+	Agent     ProwJobAgent `json:"agent,omitempty"`
+	Cluster   string       `json:"cluster,omitempty"`
+	Job       string       `json:"job,omitempty"`
+	Refs      *Refs        `json:"refs,omitempty"`
+	ExtraRefs []*Refs      `json:"extra_refs,omitempty"`
+
+	TimeoutMinutes int    `json:"timeout_minutes,omitempty"`
 	Report         bool   `json:"report,omitempty"`
 	Context        string `json:"context,omitempty"`
 	RerunCommand   string `json:"rerun_command,omitempty"`
 	MaxConcurrency int    `json:"max_concurrency,omitempty"`
 
+	Decorate         bool             `json:"decorate,omitempty"`
```

Comment:
godoc; we should have for all API fields but let's document the new ones introduced here at least